ci/fix_gitignore.sh

```diff
@@ -13,10 +13,10 @@ source ci/ci_helpers.sh
 # 2. Add the desired extension at the end
 # 3. Add a / character at the beginning of each line to conform to .gitignore file format
 #
-# e.g. multi_copy.source -> /multi_copy.sql
+# e.g. tablespace.source -> /tablespace.sql
 ls -1 src/test/regress/input | sed -E "s#(.*)\.source#/\1.sql#" > src/test/regress/sql/.gitignore
 
-# e.g. multi_copy.source -> /multi_copy.out
+# e.g. tablespace.source -> /tablespace.out
 ls -1 src/test/regress/output | sed -E "s#(.*)\.source#/\1.out#" > src/test/regress/expected/.gitignore
```

Comment:
I had several reasons:
- I wanted to remove all references to the file that is now renamed.
- I wanted to touch this script, so that it is a reminder to me to actually remove some lines from here.

After I remove all the `.source` files, I want to remove the following:
- the checks here on this file
- `input`, `output` directories
- relevant `.gitignore` rules that are generated by this script

I might as well left this file as is, or I could have added some comments with possible future work that I described here.